Code/Engine/Foundation/Math/Easing.h

```diff
@@ -0,0 +1,147 @@
+#pragma once
+
+#include <Foundation/Basics.h>
+#include <Foundation/Math/Declarations.h>
+
+namespace ezMath
+{
+  enum ezEasingFunctions
+  {
+    InLinear,
+    OutLinear,
+    InSine,
+    OutSine,
+    InOutSine,
+    InQuad,
+    OutQuad,
+    InOutQuad,
+    InCubic,
+    OutCubic,
+    InOutCubic,
+    InQuartic,
+    OutQuartic,
+    InOutQuartic,
+    InQuintic,
+    OutQuintic,
+    InOutQuintic,
+    InExpo,
+    OutExpo,
+    InOutExpo,
+    InCirc,
+    OutCirc,
+    InOutCirc,
+    InBack,
+    OutBack,
+    InOutBack,
+    InElastic,
+    OutElastic,
+    InOutElastic,
+    InBounce,
+    OutBounce,
+    InOutBounce
+  };
+
+  typedef double (*ezEasingFunction)(double);
```

Comment:
In New code we prefer "using" rather than typedef.

Reply:
Removed the definition.